extensions/cpsection/webaccount/web_service.py

```diff
@@ -16,7 +16,7 @@
 
 class WebService():
     def get_icon_name(self):
-        raise "Not implemented"
+        raise Exception("Not implemented")
```

Comment:
@walterbender, is the Web Services control panel section useful?  It doesn't seem to do anything at the moment.  If you click on it, you get "No web services are installed".  The [installer activity](http://activities.sugarlabs.org/en-US/sugar/addon/4723) hasn't been updated since 2013 and isn't on Sugar Labs GitHub but is in @i5o's GitHub https://github.com/i5o/install-extensions